<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>248</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/settings_32.png</normaloff>:/images/settings_32.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">text-align: left;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="5">
    <widget class="QTableWidget" name="tableWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Url</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enoding</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>removeRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>moveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>moveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>appendRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>editRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeRecord()</slot>
  <slot>moveDown()</slot>
  <slot>moveUp()</slot>
  <slot>appendRecord()</slot>
  <slot>editRecord()</slot>
 </slots>
</ui>
